apps/api/src/bootstrap.ts

```diff
@@ -59,6 +59,7 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   app.useGlobalPipes(
     new ValidationPipe({
       transform: true,
+      forbidUnknownValues: false,
```

Comment:
As mentioned above, there is an existing open issue regarding mapped types in nestjs, and due to us not affected by the vulnerability because of the use of class transformers, added this here.